lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && err) {
```

Comment:
I'm still genuinely curious why you are checking argument length.
If `arguments.length === 0`, isn't `err` going to be falsy no matter what?